packages/types/src/index.ts

```diff
@@ -126,6 +126,7 @@ export type Choice = {
   image?: string
 }
 
-export const EXISTING_BULK_COUPON = 'EXISTING_BULK_COUPON' as const
-export const NEW_BULK_COUPON = 'NEW_BULK_COUPON' as const
-export const NEW_INDIVIDUAL_PURCHASE = 'NEW_INDIVIDUAL_PURCHASE' as const
+export const EXISTING_BULK_COUPON = 'EXISTING_BULK_COUPON'
+export const NEW_BULK_COUPON = 'NEW_BULK_COUPON'
+export const NEW_INDIVIDUAL_PURCHASE = 'NEW_INDIVIDUAL_PURCHASE'
+export const INDIVIDUAL_TO_BULK_UPGRADE = 'INDIVIDUAL_TO_BULK_UPGRADE'
```

Comment:
Add new `INDIVIDUAL_TO_BULK_UPGRADE` type alongside the others. And removing the unnecessary `as const` casting.

There was a comment left on these lines in another PR about making them an enum -- I curious to hear more about that, I haven't used enums at all up to this point. I'd prefer to explore that outside this PR tho so as to avoid getting sidetracked from deploying the fix to the post-purchase page.